controllers/storagecluster/provider_server.go

```diff
@@ -434,3 +462,31 @@ func RandomString(l int) string {
 
 	return string(bytes)
 }
+
+func createOnboardingJob(instance *ocsv1.StorageCluster) *batchv1.Job {
+
+	// Annotation template.alpha.openshift.io/wait-for-ready ensures template readiness
+	annotations := map[string]string{"template.alpha.openshift.io/wait-for-ready": "true"}
```

Comment:
why is this annotation required?

Reply:
Forgot to remove this. It's not required now.